build-image/openhabian.pi-raspios64beta.conf

```diff
@@ -0,0 +1,59 @@
+# Modify your openHABian settings
+# Handle with care, only touch if you need to!!
+
+hostname=openhab
+username=openhabian
+userpw=openhabian
+
+# Language and time. Be sure to set these to valid values
+timeserver=0.pool.ntp.org
+locales="en_US.UTF-8 de_DE.UTF-8"
+system_default_locale="en_US.UTF-8"
+#timezone=Europe/Berlin
+
+# Wi-Fi settings. An ethernet connection is recommended
+# Requirement: RPi4, RPi3, RPi0W or a supported external Wi-Fi dongle
+# Fill in your SSID and password below, leave empty to use ethernet
+# ATTENTION: you need to escape these special characters: $, `, ", \, (newline)
+# 'Escaping' means to put an additional \ in front of that character
+wifi_ssid=""
+wifi_psk=""
+
+# Wi-Fi setting. Select a two-letter country code suitable for your location
+# E.g. US (default), DE, AU, NZ...
+# You may infringe on local legislature otherwise
+# See https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 or /usr/share/zoneinfo/zone.tab
+wifi_country=""
+
+# Eventually disable all IPv6 e.g. on installation problems
+# values: "enable", "disable"
+ipv6=enable
+
+# repo to clone from (if not master for build and test)
+repositoryurl=https://github.com/openhab/openhabian.git
+clonebranch=stable
+
+# debug verbosity mode
+# off, on (verbose output in log) or maximum (show every command)
+# as long as image is still beta, default to on
+debugmode=on
+
+# Hardware
+# pi0, pi0w, pi1, cm1, pi2, pi3, cm3, pi3+, cm3+, pi4
+#hw=pi3
+
+# Hardware architecture
+# x86_64, amd64, armv6l, armv7l, aarch64, arm64
+#hwarch=armv7l
+
+# OS Release
+# ubuntu, raspbian, debian
+# stretch, buster
+# bionic, focal
+#release=buster
+
+# Java architecture mode
+# Valid arguments: "Adopt11", "Zulu8-32", "Zulu8-64", "Zulu11-32", or "Zulu11-64"
+java_opt=Zulu11-64
```

Comment:
Java 11 default? Are you planning on this being the openHAB 3 recommended setup for a RPi3+?

Reply:
See above, zulu8-64 did not work in my last test. This might have changed in the meantime, since java installation improved since then. Not tested yet.